Restructure Build apps: add an Arbitrum essentials section (#3332)
OffchainLabs/arbitrum-docs: docs/arbitrum-essentials/bridging/configure-token-gateway/generic-custom.mdx

You can bridge tokens between the parent and child chain using the origin parent chain token and the custom token deployed on the child chain, along with the router and gateway contracts from each layer.

> Suppose you want to see an example of bridging a token from the parent to the child chain using Arbitrum’s SDK. In that case, you can check out [How to bridge tokens via Arbitrum’s standard **ERC-20** gateway](/arbitrum-essentials/bridging/deposit/tokens.mdx), specifically in Steps 2-5.

Author: ```suggestion
For an example of bridging a token from the parent to the child chain using Arbitrum's SDK, check out [How to bridge tokens via Arbitrum's standard **ERC-20** gateway](/arbitrum-essentials/bridging/deposit/tokens.mdx), specifically Steps 2-5.
```
